swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0, age = None):
+        if not isinstance(age, int):
+            self.age = 0
```

Comment:
Nice! For argument's sake, how might we use a ternary operator (like on line 7) to shorten this to one line?